### PR TITLE
More PSR-12 changes

### PR DESCRIPTION
**Pull request by michaelcullum**
Numerous changes for PSR-12, including swapping myself out.

Could I have a merge ASAP please @samdark @KorvinSzanto

### CODE REVIEW

File: proposed/extended-coding-style-guide.md [hunk 1]
```diff
@@ -107,14 +107,15 @@ There MUST NOT be more than one statement per line.
...
+tabs for indenting.
 
-### 2.5 Keywords and True/False/Null
+### 2.5 Keywords and Types
 
 PHP [keywords][] MUST be in lower case.
 
 The PHP types and keywords `int`, `true`, `object`, `float`, `false`, `mixed`,
-`bool`, `null`, `numeric`, `string` and `resource` MUST be in lower case
+`bool`, `null`, `numeric`, `string`, `void` and `resource` MUST be in lower case.
```
Comment (GrahamCampbell) [inline proposed/extended-coding-style-guide.md]: Isn't `array` missing from this list?

Comment (GrahamCampbell) [inline proposed/extended-coding-style-guide.md]: Also, what about supporting `scalar` too? I don't see it about much, but php does have an `is_scalar` method.

Comment (samdark) [inline proposed/extended-coding-style-guide.md]: There's no `scalar` keyword as far as I know. `array` should be there, correct.

Comment (samdark) [inline proposed/extended-coding-style-guide.md]: https://github.com/php-fig/fig-standards/commit/2b9d9c17ccdcfae221613084d0925e628de8d7c4

**Pull request merged**